terraform/modules/qms/main.tf

```diff
@@ -575,6 +576,16 @@ resource "google_logging_project_sink" "instance-sink" {
   depends_on             = [module.project-services]
 }
 
+resource "null_resource" "update_logging_sink" {
+  triggers = {
+    always_run = "${timestamp()}"
+  }
+
+  provisioner "local-exec" {
+    command = "gcloud logging sinks update _Default --log-filter=\"NOT LOG_ID('cloudaudit.googleapis.com/access_transparency') AND NOT LOG_ID('externalaudit.googleapis.com/access_transparency') AND NOT (severity=DEFAULT OR severity=DEBUG AND resource.labels.function_name=quotaMonitoringListProjects) AND NOT (severity=DEFAULT OR severity=DEBUG AND resource.labels.function_name=quotaMonitoringScanProjects)\""
```

Comment:
Why is access transparency included in the exclusion or some examples of what would be excluded by these conditions?